news/_posts/public-oss-demo/2022-09-08-public-demo-6-2022.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "PrestaShop Project Public Demo - September 2022 session"
+subtitle: "We held the sixth session of this year open source project's public demo"
+date:   2022-09-08 11:00
+authors: [ PrestaShop ]
+icon: icon-video-camera
+tags: [contribute, event, community]
+---
+
+The project's sixth session of this year edition of our public demo was held and [streamed live](https://www.youtube.com/watch?v=R6fwb7FiiCI) on September 7th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=R6fwb7FiiCI) Beginning of the session, the introduction
+- [02:02](https://youtu.be/R6fwb7FiiCI?t=122) Project news: PrestaShop 1.7.8.7 release
```

Comment:
Do you create the timestamps manually like I did? 😝  maybe you want to script it

Reply:
yeah, fortunately, it's quite fast. A huge time-saver is that if you paste it to the YT description, it creates proper segments in the video automatically :D So a few minutes here is not a problem :D